src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/af-page/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Template">
+    <jcr:content
+        jcr:description="Generic template for an Adaptive Form."
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Adaptive Form"/>
```

Comment:
@sakshi-arora1 - There is no jcr:lastmodified on template or policy. AFAIR its mandatory otherwise Adaptive form will be never cached in-memory.
CC - @rismehta

Reply:
added jcr:lastmodified